ansible/roles/fanout/templates/arista_7060_deploy.j2

```diff
@@ -21,101 +21,33 @@ no spanning-tree vlan {{ device_vlan_range[inventory_hostname] | list | join(','
-   no shutdown
-{%     elif device_conn[inventory_hostname][intf]['speed'] == "40000" %}
-   description {{ device_conn[inventory_hostname][intf]['peerdevice'] }}-{{ device_conn[inventory_hostname][intf]['peerport'] }}
-   switchport access vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
-   switchport mode dot1q-tunnel
-   spanning-tree portfast
-   speed forced 40gfull
-   no switchport mac address learning
-   no shutdown
-{%     elif device_conn[inventory_hostname][intf]['speed'] == "50000" %}
-   description {{ device_conn[inventory_hostname][intf]['peerdevice'] }}-{{ device_conn[inventory_hostname][intf]['peerport'] }}
-   switchport access vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
-   switchport mode dot1q-tunnel
-   spanning-tree portfast
-   speed forced 50gfull
-   no error-correction encoding
-   no switchport mac address learning
-   no shutdown
-{% set intf =  'Ethernet' + i|string + '/3'  %}
-interface {{ intf }}
-{%         if intf in device_port_vlans[inventory_hostname] and device_port_vlans[inventory_hostname][intf]['mode'] != "Trunk" %}
-   description {{ device_conn[inventory_hostname][intf]['peerdevice'] }}-{{ device_conn[inventory_hostname][intf]['peerport'] }}
-   switchport access vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
-   switchport mode dot1q-tunnel
-   spanning-tree portfast
-   speed forced 50gfull
-   no error-correction encoding
-   no switchport mac address learning
-   no shutdown
-{%         else %}
-   shutdown
-{%         endif %}
-{%     elif device_conn[inventory_hostname][intf]['speed'] == "10000" %}
-   description {{ device_conn[inventory_hostname][intf]['peerdevice'] }}-{{ device_conn[inventory_hostname][intf]['peerport'] }}
-   switchport access vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
+   speed force {{ device_conn[inventory_hostname][intf]['speed'] }}full
+{%   if device_port_vlans[inventory_hostname][intf]['mode'] == 'Trunk' %}
+   switchport mode trunk
+   switchport trunk allowed vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
+{%   else %}
    switchport mode dot1q-tunnel
-   spanning-tree portfast
-   speed forced 10gfull
-   no error-correction encoding
-   no switchport mac address learning
-   no shutdown
-{%         for j in range(2,5) %}
-{% set intf =  'Ethernet' + i|string + '/' + j|string %}
-interface {{ intf }}
-{%             if intf in device_port_vlans[inventory_hostname] and device_port_vlans[inventory_hostname][intf]['mode'] != "Trunk" %}
-   description {{ device_conn[inventory_hostname][intf]['peerdevice'] }}-{{ device_conn[inventory_hostname][intf]['peerport'] }}
    switchport access vlan {{ device_port_vlans[inventory_hostname][intf]['vlanids'] }}
-   switchport mode dot1q-tunnel
-   spanning-tree portfast
-   speed forced 10gfull
-   no error-correction encoding
-   no switchport mac address learning
-   no shutdown
-{%             else %}
-   shutdown
```

Comment:
How to ensure that disabled or unconnected ports are in `shutdown` state?

Reply:
Currently, we didn't see any issues caused by removing this config. But not sure if there exists any potential issues. @yxieca can you please help to check it? Thanks!

Reply:
@lizhijianrd, is EOS port by default shutdown?

Reply:
@yxieca Previously, I removed the default shutdown because it's hard to iterate all the unconnected ports and config them as shutdown (Current code in the repo is not correct, and I'm opening this PR to fix it), and these unconnected ports can be considered as "shutdown physically". 
I think your concern is right, EOS port is not by default shutdown, and we should have some precaution mechanism. So I added the following lines to Arista-7060 startup-config template in my latest commit:

```
interface defaults
   ethernet
      shutdown
```

This will config `shutdown` to all the ports without explicit configuration. I verified it in our local lab, and the `show running-config` give the following output:

```
# show running-config
... 
interface Ethernet1/3
   description <dut-hostname>-<dut-port>
   no shutdown
   speed forced 10000full
   no error-correction encoding
   switchport access vlan 251
   switchport mode dot1q-tunnel
   no switchport mac address learning
   spanning-tree portfast
!
interface Ethernet1/4
   description <dut-hostname>-<dut-port>
   no shutdown
   speed forced 10000full
   no error-correction encoding
   switchport access vlan 252
   switchport mode dot1q-tunnel
   no switchport mac address learning
   spanning-tree portfast
!
interface Ethernet2/1
   description <dut-hostname>-<dut-port>
   no shutdown
   speed forced 100gfull
   error-correction encoding reed-solomon
   switchport access vlan 316
   switchport mode dot1q-tunnel
   no switchport mac address learning
   spanning-tree portfast
!
interface Ethernet2/2
   shutdown
!
interface Ethernet2/3
   shutdown
!
interface Ethernet2/4
   shutdown
!
interface Ethernet3/1
   shutdown
!
...
```

Reply:
I have no further concerns. @wangxin ?

Reply:
I am happy with adding the default shutdown config.